dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClasspathChecker.java

```diff
@@ -125,10 +109,11 @@ public static void main(String[] arguments)
   public ClasspathCheckReport findLinkageErrors() {
     // Validate linkage error of each reference
     ImmutableList.Builder<JarLinkageReport> jarLinkageReports = ImmutableList.builder();
-    for (Map.Entry<Path, SymbolReferenceSet> entry : jarToSymbols.entrySet()) {
-      Path jarPath = entry.getKey();
-      jarLinkageReports.add(generateLinkageReport(jarPath, entry.getValue()));
-    }
+
+    jarToSymbols.forEach(
+        (jar, symbolReferenceSet) -> {
```

Comment:
I think you can use an expression lambda here. (Remove the braces and the semicolon.)

Reply:
Removed braces.